src/corelibs/U2View/src/ov_phyltree/GraphicsBranchItem.h

```diff
@@ -98,7 +98,8 @@ class U2VIEW_EXPORT GraphicsBranchItem : public QAbstractGraphicsShapeItem {
 
     void updateSettings(const OptionsMap &settings);
     void updateChildSettings(const OptionsMap &settings);
-    void updateTextSettings(const QFont &font, const QColor &color);
+    void updateTextFont(const QFont &font);
```

Comment:
Please add a minimal doc for all modified/added methods.
The doc format is optional, you can check mine in the IOAdapterTextStream.h file.
I usually start the method doc with a verb (what this method does) and include only non-trivial information into the doc.